config/database.yml

```diff
@@ -1,33 +1,23 @@
-development:
+defaults: &defaults
   adapter: postgresql
   encoding: unicode
-  database: open_food_network_dev
   pool: 5
   host: localhost
-  username: ofn
-  password: f00d
+  username: <%= ENV.fetch('DB_USERNAME', 'ofn') %>
```

Comment:
Can we call it `OFN_DB_USERNAME` and `OFN_DB_PASSWORD`? I'm just imagining other apps on my computer using the same variable names.

Reply:
Prefixes added :+1: